src/rpc/rawtransaction.cpp

```diff
@@ -555,6 +557,7 @@ static UniValue decodescript(const JSONRPCRequest& request)
                             {
                                 {RPCResult::Type::STR, "address", "segwit address"},
                             }},
+                            {RPCResult::Type::STR_HEX, "desc", "Inferred descriptor for the output\n"},
```

Comment:
why hex?

Reply:
it's not, forgot to change this when fixing the other one